app/views/organizations/public_faq/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<header class="pt-5 pb-5" >
+  <div class="container">
+    <div class="text-center">
+      <h1 class="section-heading text-uppercase underline">
+        <%= t('.header') %>
+      </h1>
+    </div>
+  </div> <!--container-->
+</header>
+
+<section class="pb-5" id="public_faq">
+  <div class="container">
+    <div class="row justify-content-center">
+      <div class="col-lg-8 col-md-10 col-12">
+        <div class="accordion accordion-flush" id="accordionExample">
+          <% @faqs.each_with_index do |faq, index| %>
```

Comment:
This all works well, just wondering when the index would come in to play. If index is not needed, could we simplify this and render as a collection `render partial: "faq", collection: @faqs`. The new `public_faq/_faq` partial could then use the dom_id helper. `id="<%= dom_id(faq, :heading) %>"`

Other small nitpick, maybe change "accordionExample" and "#accordionExample" with something more specific.

Reply:
@jmilljr24 yes rendering partials would make more sense. The index is required for the JS to work so that each FAQ Q and A id matches but is different from the rest. I have not see the dom_id helper but obviously there is something like this as it's a common use case. Good catch!

Reply:
Sorry I should have highlighted this line `<% faq_id = "faq_#{faq.id || index + 1}" %>`. The faq should always have an id so the `index + 1` would ever be used.

Reply:
Yes I see. Haha I really biffed this one. I rushed it. I will fix it.

Reply:
@jmilljr24 updated to use a question and answer component that we can use elsewhere.